qtconsole/tests/test_console_widget.py

```diff
@@ -415,35 +417,140 @@ def is_complete(self, source):
         w.execute(interactive=True)
         assert responses == [('complete', None)]
 
-    def test_debug(self):
-        """
-        Make sure the cursor works while debugging 
-        
-        It might not because the console is "_executing"
-        """
-        # Kernel client to test the responses of is_complete
-        class TestIPyKernelClient(object):
-            def is_complete(self, source):
-                tm = TransformerManager()
-                check_complete = tm.check_complete(source)
-                responses.append(check_complete)
 
-        # Initialize widget
-        responses = []
-        w = ConsoleWidget()
-        w._append_plain_text('Header\n')
-        w._prompt = 'prompt>'
-        w._show_prompt()
-        w.kernel_client = TestIPyKernelClient()
-        control = w._control
+@pytest.fixture
+def qtconsole():
```

Comment:
Please move this fixture to the beginning of the file, make it depend on `qtbot` and use `qtbot` as in the file I pointed you to.